core/lib/multivm/src/versions/vm_fast/tests/mod.rs

```diff
@@ -78,7 +81,11 @@ impl PartialEq for VmStateDump {
     }
 }
 
-impl TestedVm for Vm<ImmutableStorageView<InMemoryStorage>> {
+impl<E: Tracer + Default, V: ValidationMode> TestedVm
+    for Vm<ImmutableStorageView<InMemoryStorage>, WithBuiltinTracers<E, V>>
```

Comment:
Nit: You can create an alias for this type, similarly to [`TestedLatestVm`](https://github.com/matter-labs/zksync-era/blob/98803effbca2d285cf600dbaeb78818f56b25951/core/lib/multivm/src/versions/vm_latest/tests/mod.rs#L64).